client/src/components/Header.js

```diff
@@ -1,23 +1,15 @@
 import React from "react";
 import AppBar from "@mui/material/AppBar";
 import styled from "styled-components";
-// import Container from "@mui/material/Container";
 import "../App.css";
 
-const Container = styled.h1`
- 
-  font-size: 1.5em;
-  text-align: center;
+const Container = styled.header`
+  h1 {
+    font-size: 1.5em;
+    text-align: center;
```

Comment:
I think this is a regression and will break the existing implementation.  Can it be set back to the `h1` tag?

Reply:
Yes reset

Reply:
Working on converting patch before re-commiting changes